common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -508,3 +508,65 @@ def test_studio_settings_page_a11y(self):
         })
 
         self.settings_page.a11y_audit.check_for_accessibility_errors()
+
+
+class AdvancedSettingsTest(StudioCourseTest):
+    """
+    Tests for Advanced Settings tab in studio.
+    """
+    def setUp(self):  # pylint: disable=arguments-differ
```

Comment:
@benpatterson If we see the setUp function of StudioCourseTest we see there are two more arguments apart from self. We can change the value of arguments of setUp function from setUp(self) to setUp(self, is_staff=False, test_xss=True). There are many instances in the file where we could have used same arguments as parent class but we didn't and instead we used to ignore it.  This is the reason why I ignored it too. I thought there was some consideration behind it.